docker-compose.dev.yml

```diff
@@ -0,0 +1,27 @@
+version: "3"
+
+services:
+  mindwendel:
+    container_name: mindwendel
+    build:
+      context: .
+      target: development
+    environment:
+      DATABASE_USER: postgres
+      DATABASE_USER_PASSWORD: postgres
+      DATABASE_NAME: mindwendel
+      DATABASE_PORT: 5432
+      DATABASE_HOST: db
+    ports:
+      - "4000:4000"
+    depends_on:
+      - db
```

Comment:
db isnt included in this file, or is it?

Reply:
I removed this file.